drivers/interrupt_controller/intc_gic.c

```diff
@@ -126,10 +137,15 @@ static void gic_dist_init(void)
 	sys_write32(0, GICD_CTLR);
 
 	/*
-	 * Set all global interrupts to all CPUs.
+	 * Enable all global interrupts distributing to CPUs listed
+	 * in dts with the count of CONFIG_MP_NUM_CPUS.
 	 */
+	for (i = 0; i < CONFIG_MP_NUM_CPUS; i++)
+		cpu_mask |= BIT(cpu_mpid_list[i]);
```

Comment:
Nit: as per the coding guideline
```suggestion
	for (i = 0; i < CONFIG_MP_NUM_CPUS; i++) {
		cpu_mask |= BIT(cpu_mpid_list[i]);
	}
```

Reply:
Done, thanks.